### ISSUE TITLE
Can't add or remove events programmatically on v4

### ISSUE THREAD

**Issue by zellb** [StickNitro/ngx-fullcalendar#10]
When trying to add new event or remove a current event from calendar using array's push or splice method Im getting the following error: 

```
TypeError: this.calendar.removeEventSources is not a function
    at FullCalendarComponent.push../node_modules/ngx-fullcalendar/fesm5/ngx-fullcalendar.js.FullCalendarComponent.ngDoCheck (ngx-fullcalendar.js:230)
    at checkAndUpdateDirectiveInline (core.js:22004)
    at checkAndUpdateNodeInline (core.js:23265)
    at checkAndUpdateNode (core.js:23227)
    at debugCheckAndUpdateNode (core.js:23861)
    at debugCheckDirectivesFn (core.js:23821)
    at Object.eval [as updateDirectives] (SchedulersComponent.html:2)
    at Object.debugUpdateDirectives [as updateDirectives] (core.js:23813)
    at checkAndUpdateView (core.js:23209)
    at callViewAction (core.js:23450)
```
```
 this.events.push({
                id: 'e',
                title: 'Friends coming round 2',
                start: '2018-07-29T18:00:00',
                end: '2018-07-29T23:00:00'
            }
       );
```

**Comment by martinsiden** [comment]
That's because that method has been removed in v4 of Fullcalendar. If you look at this page: https://fullcalendar.io/docs/v4/release-notes it clearly states that:

> removeEventSources | Removed. Instead, use multiple EventSource::remove calls. Group together for performance with batchRendering.

So you should fetch all events that the calendar currently has in memory, and then call remove on each of them. Once that's done, you can add new events using the `addEvent` method on fullcalendar.

**Comment by martinsiden** [comment]
Try something like this:

`@ViewChild('theCalendar') theCalendar: FullCalendarComponent;

// Grab all events
    let allevents = this.theCalendar.calendar.getEvents();

    // Loop all events
    allevents.forEach(el => {
      
      // And remove them
      el.remove();

    });

// Loop that array
    events.forEach(el => {

      // And add them to the calendar again
      this.theCalendar.calendar.addEvent(el);

    });`

**Comment by StickNitro** [comment]
Fixed in v5.0.0-alpha.1

**Comment by CrackerakiUA** [comment]
@martinsiden please check https://github.com/StickNitro/ngx-fullcalendar/issues/21
